lib/ex_unit/lib/ex_unit/filters.ex

```diff
@@ -97,7 +99,7 @@ defmodule ExUnit.Filters do
     end
   end
 
-  defp has_tag({:line, line}, tags, collection) do
+  defp has_tag({:line, line}, %{line: _} = tags, collection) do
```

Comment:
I had to add this to avoid matching on 85 above. This version of has_tag assumes `:line` is in `tags`.